docs/guide/built-in-rules-callback.md

```diff
@@ -53,8 +53,7 @@ new Callback(
 
 ### Value validation
 
-`Callback` rule can be used to add validation missing in built-in rules for a single attribute's value. Below is the 
-example verifying that a value is a valid [YAML] string (additionally requires `yaml` PHP extension):
+Below is an example that verifying that a value is a valid [YAML] string (additionally requires the `yaml` PHP extension):
```

Comment:
Missing first sentence which was added recently during review fixes. I think it has value and suggest to keep it. Or otherwise please explain why it was deleted.

Reply:
Related - https://github.com/yiisoft/validator/pull/507#discussion_r1088299791. Probably was removed during resolving merge conflicts.